tests/console/journalctl.pm

```diff
@@ -236,6 +237,7 @@ sub run {
     write_test_log_entries \%log_entries;
     assert_test_log_entries(\%log_entries, \@boots);
     # verify sealing
+    sleep 10;
```

Comment:
Preferably, I would suggest putting the value into the constant hash and update seal setup invoke to use this constant, in such a case it is clear why the test waits for a certain amount of time.

Reply:
:+1: